reference/5.0/Microsoft.PowerShell.Core/Import-Module.md

```diff
@@ -910,18 +1055,22 @@ Accept wildcard characters: False
 ```
 
 ### -Scope
+
 Specifies a scope into which this cmdlet imports the module.
 
 The acceptable values for this parameter are:
 
-- **Global**. Available to all commands in the session. Equivalent to the *Global* parameter.
+- **Global**. Available to all commands in the session. Equivalent to the **Global** parameter.
 - **Local**. Available only in the current scope.
 
-By default, when Import-Module cmdlet is called from the command prompt, script file, or scriptblock, all the commands are imported into the global session state.
-You can use the **-Scope** parameter with the value of **Local** to import module content into the script or scriptblock scope.
+By default, when `Import-Module` cmdlet is called from the command prompt, script file, or
+scriptblock, all the commands are imported into the global session state. You can use the **-Scope**
```

Comment:
Use backticks with `-Scope` or bold with **Scope**.

Same for Global and Scope Global.